utility/Vector.cpp

```diff
@@ -17,6 +17,12 @@ Vector3 Vector3::CrossProduct(const Vector3& a, const Vector3& b)
                    a.X * b.Y - a.Y * b.X);
 }
 
+float Vector3::GetDistance(const Vector3& a, const Vector3& b)
```

Comment:
Can you turn this into not static and taking only one other vector?  Also, would you mind searching for where this calculation is done elsewhere and use this function instead?